docs/docusaurus.config.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
     },
     navbar: {
       title: 'Chart.js Annotation Plugin',
+      items: [
+        {href: './samples', label: 'Samples'},
```

Comment:
Will this automatically be pulled into the deploy? I think you might need something in the CI to copy the samples folder to the static dir

Reply:
The `npm run samples` in root directory builds the samples to the static folder, works locally.
Its added in the documentation action.

Reply:
Ah, got it. Missed that it built directly there